samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -219,10 +221,17 @@ public async Task<SKContext> ChatAsync(string message, SKContext context)
             return context;
         }
 
+        // Retrieve the prompt used to generate the response
+        // and return it to the caller via the context variables.
+        var prompt = chatContext.Variables.ContainsKey("prompt")
+            ? chatContext.Variables["prompt"]
+            : string.Empty;
+        context.Variables.Set("prompt", prompt);
```

Comment:
Why would the prompt be empty right here? On failure?

Reply:
In case the context variables do not contain the prompt, we set it to empty instead of raising an error.